ubuntu-headless-qt/include/05_create_wic.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# --------------------------------------------------------------------------#
+# function create_wic use to create wic from ubuntu rootfs.
```

Comment:
This description looks like a function description. Can you update it and move the old description to the correct location of the mentioned functions?